bsp_diff/caas/device/intel/civ/host/vm-manager/0046-Multi-Android-VM-support.patch

```diff
@@ -0,0 +1,382 @@
++        done
++}
++
++read_args "$@"
++
+ copy_civ_ini
+ setup_civ_ini
+ 
+diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
+index 0c1e0a1..e104419 100755
+--- a/scripts/setup_host.sh
++++ b/scripts/setup_host.sh
+@@ -19,6 +19,10 @@ CIV_WORK_DIR=$(pwd)
+ CIV_GOP_DIR=$CIV_WORK_DIR/GOP_PKG
+ CIV_VERTICAl_DIR=$CIV_WORK_DIR/vertical_patches/host
+ VM_MANAGER_VERSION=v1.2.3
++VSOCK_ID=3
++VM_NAME=Android-CIV1
++ADB_PORT=5555
++USER_HOME=$(getent passwd $SUDO_USER | cut -d: -f6)
+ 
+ #---------      Functions    -------------------
+ function error() {
+@@ -575,6 +579,11 @@ function parse_arg() {
+                 setup_remote_infer || return -1
+                 ;;
+ 
++            -n)
++                VM_NAME=$2
++		shift
++                ;;
++
+             --auto-start)
+                 install_auto_start_service "$2" || return -1
+                 shift
+@@ -599,13 +608,40 @@ function parse_arg() {
+ }
+ 
+ function setup_civ_ini() {
+-        $CIV_WORK_DIR/scripts/setup_civ_ini.sh
++        $CIV_WORK_DIR/scripts/setup_civ_ini.sh -v $1 -p $2 -n $3
++}
++
++function create_vm_dir() {
++        if [ -d $CIV_WORK_DIR/$VM_NAME ]
++        then
++                echo "Folder with name $VM_NAME already present. Delete and create new folder? Please enter yes/no"
++		read input
++		if [ $input = "yes" ]; then
++	                rm -rf $CIV_DIR/$VM_NAME
++	        else
++	                exit
++	        fi
++        fi
++        echo "Creating Dir: $CIV_WORK_DIR/$VM_NAME"
++        mkdir $CIV_WORK_DIR/$VM_NAME
+ }
+ 
+-function create_aaf_dir() {
+-        mkdir -p $CIV_WORK_DIR/scripts/aaf
+-}
++function copy_files_for_vm() {
++    echo "Copying file: $CIV_WORK_DIR/$VM_NAME"
++    mkdir -p $CIV_WORK_DIR/$VM_NAME/scripts/aaf
++
++    req_files=("$CIV_WORK_DIR/OVMF.fd"
++                   "$CIV_WORK_DIR/scripts/rpmb_dev")
++    for file in ${req_files[@]}; do
++        if [ ! -f $file ]; then
++            echo "Error: $file file is missing"
++            exit -1
++        fi
++    done
+ 
++    cp $CIV_WORK_DIR/OVMF.fd $CIV_WORK_DIR/$VM_NAME/
++    cp $CIV_WORK_DIR/scripts/rpmb_dev $CIV_WORK_DIR/$VM_NAME/scripts/
++}
+ 
+ #-------------    main processes    -------------
+ 
+@@ -623,10 +659,11 @@ ubu_install_qemu_gvt
+ ubu_build_ovmf_gvt
+ ubu_enable_host_gvt
+ ubu_update_fw
+-
+ install_vm_manager
+-create_aaf_dir
+-setup_civ_ini
++
++create_vm_dir
++setup_civ_ini $VSOCK_ID $ADB_PORT $VM_NAME
++copy_files_for_vm
+ 
+ prepare_required_scripts
+ ubu_install_swtpm
+diff --git a/scripts/setup_multi_civ_vm.sh b/scripts/setup_multi_civ_vm.sh
+new file mode 100755
+index 0000000..5359d8d
+--- /dev/null
++++ b/scripts/setup_multi_civ_vm.sh
+@@ -0,0 +1,141 @@
++#!/bin/bash
++
++# Copyright (c) 2023 Intel Corporation.
++# All rights reserved.
++#
++# SPDX-License-Identifier: Apache-2.0
++
++set -eE
++
++#---------      Global variable     -------------------
++CIV_DIR=$(pwd)
++USER_HOME=$(getent passwd $SUDO_USER | cut -d: -f6)
++VM_NAME=Android-CIV
```

Comment:
Standardise and use Android-CIV1?

Reply:
Kept it as Android-CIV as numbers are appended to Android-CIV.